tests/performance/loadtester.py

```
@@ -1,13 +1,14 @@
-import eventlet
-eventlet.monkey_patch()
+from gevent import monkey
+monkey.patch_all()
+from gevent.pool import Pool  # noqa: E402
 from secator.runners import Workflow, Task, Scan  # noqa: E402
 from secator.template import TemplateLoader  # noqa: E402
 from secator.celery import *  # noqa: E402,F403
```

Flake8 (7.3.0)

[error] 'secator.celery.*' imported but unused

(F401)